packages/mapviewer/src/modules/infobox/components/FeatureDetail.vue

```diff
@@ -6,13 +6,13 @@ import { useI18n } from 'vue-i18n'
 import { useStore } from 'vuex'
 
 import SelectableFeature from '@/api/features/SelectableFeature.class.js'
-import { WHITELISTED_HOSTNAMES } from '@/config/staging.config'
+import { BLOCKED_EXTENSIONS, WHITELISTED_HOSTNAMES } from '@/config/security.config'
 import FeatureAreaInfo from '@/modules/infobox/components/FeatureAreaInfo.vue'
 import FeatureDetailDisclaimer from '@/modules/infobox/components/FeatureDetailDisclaimer.vue'
 import CoordinateCopySlot from '@/utils/components/CoordinateCopySlot.vue'
 import allFormats from '@/utils/coordinates/coordinateFormat'
 
-const { feature } = defineProps({
+const props = defineProps({
```

Comment:
props should be deconstructed as soon as defined, see [`CONTRIBUTING.md#vue-composition-api`](https://github.com/geoadmin/web-mapviewer/blob/develop/CONTRIBUTING.md#vue-composition-api)

Reply:
ah yes, thanks!